.github/workflows/test.yml

```diff
@@ -19,3 +17,14 @@ jobs:
 
     - name: Test
       run: go test -v -race ./...
+
+  docker:
```

Comment:
Are you throwing this in as a sanity check to make sure it keeps doing successful docker builds in between releases?

Reply:
Yeah, exactly. Ideally it would test that the proxy server actually runs and handles requests, but this at least confirms that the docker build completes and the resulting binary inside can run successfully.